pkg/clients/aws/aws.go

```diff
@@ -150,5 +121,11 @@ func (c *Client) TerminateEC2Instance(ctx context.Context, instanceID string) er
 		return handledErrors.NewGenericError(err)
 	}
 
+	// Wait up to 5 minutes for the instance to be terminated
+	waiter := ec2.NewInstanceTerminatedWaiter(c)
+	if err := waiter.Wait(ctx, &ec2.DescribeInstancesInput{InstanceIds: []string{instanceID}}, 5*time.Minute); err != nil {
```

Comment:
How did we arrive at the 5 minute wait time ? Is this what the previous behavior did ?